docs/getting_started/tutorial.md

```diff
@@ -347,7 +363,7 @@ other page type.
 Publish each blog post when you are done editing.
 
 You should now have the very beginnings of a working blog.
-Access the `/blog` URL and you should see something like this:
+Access the `http://127.0.0.1:8000/blog` URL and you should see something like this:
```

Comment:
This was actually removed in a [previous PR](See previous change - https://github.com/wagtail/wagtail/pull/9299/files#diff-ae6d4a70a5209ad442f837a13f901d3e090bd84beafcfcbb9268ae9e01f31158R271), maybe we can do something better here. I agree we should keep this full IP address here but think we can improve this with a change near the top of the page.



Can you please maybe do a new change near the top of this page.

> If everything worked, <http://127.0.0.1:8000> will show you a welcome page:

Maybe we need a call out for something like 
```
   ```{note}
   Throughout this tutorial we will use `http://127.0.0.1:8000` but depending on your setup this could be `http://localhost:8000/` or a different IP address or port. Please read the console output of `manage.py runserver` if you get stuck here ad it will tell you what the IP / URL is of your site running locally.
   ```
```